src/openfermion/circuits/design_utils.py

```diff
@@ -0,0 +1,58 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+'''Tools to assist in circuit/primitive design'''
```

Comment:
I think we should avoid calling stuff `utils`/creating name filenames that contain `utils` e.t.c. Since we've merged openfermion-cirq to here, we already have `Tools to assist in circuit/primitive design`. Hence this docstring is vague. Let's also move this file to https://github.com/quantumlib/OpenFermion/tree/master/src/openfermion/circuits/trotter